server/broadcast.go

```diff
@@ -261,15 +261,13 @@ func processSegment(cxn *rtmpConnection, seg *stream.HLSSegment) {
 		}
 	}
 
-	// Process the rest of the segment asynchronously - transcode
-	go func() {
-		for true {
-			// if fails, retry; rudimentary
-			if err := transcodeSegment(cxn, seg, name); err == nil {
-				return
-			}
+	for true {
```

Comment:
Please remove `true` and change this to:
```golang
for {
   [...]
}
```

Reply:
Do we really want to try this repeatedly forever?  What if it errors consistently?  Should there be a sanity limit here, rather than loop forever?

Reply:
We retry only for as long as there is an orchestrator available in the working set. If the working set empties, `nil` is returned as a signal to exit. See the additional note within this PR about perhaps being a little more explicit about "non-retryable errors".

But as the comment following this line mentions, this is rudimentary :) How to improve the retry strategy is something that can be discussed / implemented separately outside this PR.

Reply:
I reiterate my request: Please remove `true` and change this to:
```golang
for {
   [...]
}
```

Reply:
Why does this matter? Just stylistic preference?

Reply:
We should write idiomatic code in golang.  Generally, there is no place for "style" in golang.  There is the idiomatic way, and there is everything else.   The idiomatic way to loop endlessly until internal interruption is as I described above.